src/components/apiErrorAlert/ApiErrorAlert-nb.ts

```diff
@@ -46,4 +46,13 @@ export default {
         'Kontrollsimuleringen er ulik saksbehandlers simulering',
 
     'feil.ukjentFeil': 'Ukjent feil',
+
+    'lukk.dato.ugyldig': 'Ugyldig dato',
+    'lukk.input.ugyldig': 'Ugyldig input',
+    'lukk.søknad.alleredeLukket': 'Søknaden er allerede lukket',
+    'lukk.søknad.manglerOppgave': 'Søknaden mangler oppgave',
```

Comment:
Jeg ser at denne er litt upresis:
```
is Søknad.Ny, is Søknad.Journalført.UtenOppgave -> {
                log.warn("Kan ikke lukke søknad ${søknad.id} siden den mangler oppgave. Se drifts-endepunktet /drift/søknader/fix.")
                KunneIkkeLukkeSøknad.SøknadManglerOppgave.left()
            }
```

Det ser ut som den og kan være ikke journalført også. Kanskje vi skal legge det inn i teksten med en gang og heller rydde opp bakover etterpå.

Reply:
Det er väl ikke så relevant for saksbehandlare å få melding om drifts-endepunktet? De har väl ändå ikke tillgång till det?.

Reply:
F.eks. `Søknaden er ikke journalført og/eller mangler oppgave`